### ISSUE TITLE
Fixed NPE when queue is closed

### ISSUE THREAD

**Issue by dpisklov** [OpenHFT/Chronicle-Queue#494]
```
2018-06-08T16:54:05.440+0800 [main/event-loop-monitorevent-loop-monitor] WARN MonitorEventLoop - Loop terminated due to exception
java.lang.NullPointerException
                at net.openhft.chronicle.bytes.NativeBytesStore.readVolatileLong(NativeBytesStore.java:370)
                at net.openhft.chronicle.bytes.ref.BinaryLongReference.getVolatileValue(BinaryLongReference.java:83)
                at net.openhft.chronicle.queue.impl.single.TSQueueLock.waitForLock(TSQueueLock.java:100)
                at net.openhft.chronicle.queue.impl.single.SingleChronicleQueue.acquireAppender(SingleChronicleQueue.java:374)
                at software.chronicle.services.util.MultiChronicleQueue.acquireAppender(MultiChronicleQueue.java:166)
                at software.chronicle.runner.v2.VanillaRunnerPretouchOutputController.lambda$null$0(VanillaRunnerPretouchOutputController.java:17)
                at software.chronicle.runner.v2.PeriodicUpdateEventHandler.timedAction(PeriodicUpdateEventHandler.java:33)
                at net.openhft.chronicle.threads.TimedEventHandler.action(TimedEventHandler.java:34)
                at net.openhft.chronicle.threads.MonitorEventLoop.runHandlers(MonitorEventLoop.java:137)
                at net.openhft.chronicle.threads.MonitorEventLoop.run(MonitorEventLoop.java:115)
                at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
                at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
                at java.lang.Thread.run(Thread.java:745)
```

**Comment by hft-team-city** [comment]
Released in Chronicle-Queue-4.16.4 (https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-4.16.4), BOM-1.16.32 (https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-1.16.32)

**Comment by hft-team-city** [comment]
Released in Chronicle-Queue-4.16.4 (https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-4.16.4), BOM-1.16.39 (https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-1.16.39)
